ext/web/12_location.js

```diff
@@ -378,8 +377,9 @@
     locationDescriptor: {
       get() {
         if (location == null) {
-          throw new ReferenceError(
-            `Access to "location", run again with --location <href>.`,
+          console.warn(
+            "Warning: accessing undefined 'location' global, " +
+              "run again with --location <href>.",
           );
```

Comment:
Discussed this with @lucacasonato, we will like this warning should not be printed, otherwise users feature testing if `location` is available will see these warnings.

Reply:
Ok. Let's remove